src/popover/popover.scss

```diff
@@ -0,0 +1,40 @@
+.popover-container {
+  position: relative;
+
+  & > .popover {
+    position: absolute;
+    display: none;
+
+    &.top {
+      top: 0;
+      left: 50%;
+      transform: translate(-50%, -100%);
+    }
+
+    &.bottom {
+      bottom: 0;
+      left: 50%;
+      transform: translate(-50%, 100%);
+    }
+
+    &.left {
+      bottom: 50%;
+      left: 0;
+      transform: translate(-100%, 50%);
+    }
+
+    &.right {
+      bottom: 50%;
+      right: 0;
+      transform: translate(100%, 50%);
+    }
+  }
+
+  &:hover > .popover {
```

Comment:
Tooltips should be shown on hover, but popovers should be shown on click, according to https://ux.stackexchange.com/questions/88844/when-should-i-use-a-popover-vs-a-tooltip
This is backed by both [Bootstrap](https://getbootstrap.com/docs/4.0/components/popovers/#example) and [Material](https://material-ui.com/components/popover/). To show on hover can be an option that requires a prop

Reply:
I would disagree here. Again, because in innopoints we attached the popovers to links, which had a click action of their own. I don't see any strong reason why exactly popovers should only be shown on click, so #wontfix